src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -55,6 +55,7 @@ using std::string;
 using std::map;
```

Comment:
Maybe we should include this CNR entry?

"Fix: Hardening: safer input/ouput logic for csub documents in MongoDB and cache"